src/main/java/Ui.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Scanner;
+
+public class Ui {
+    private Scanner in = new Scanner(System.in);
+    public void printlogo(){
```

Comment:
Should we have printlogo() to be in camelCase, like printLogo()?

Reply:
yea, thx for point out